src/pubsub.c

```diff
@@ -353,18 +353,34 @@ int pubsubPublishMessage(robj *channel, robj *message) {
     return receivers;
 }
 
+/* This wraps handling ACL channel permissions for the given client. */
+int pubsubCheckACLPermissions(client *c, int idx, int count, int literal) {
```

Comment:
most functions have OrReply() at the end to clarify they also respond if there is an error.

Reply:
Indeed, standardized the name.